curriculum/challenges/english/25-front-end-development/quiz-javascript-object-oriented-programming/66edd5267b52f8f617d8364f.md

```diff
@@ -17,439 +17,458 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+From the statements below, which one correctly defines OOP?
 
 #### --distractors--
 
-Placeholder distractor 1
+it is a programming model that is more concerned with functions.
 
 ---
 
-Placeholder distractor 2
+It is a programming model that is focused on the logic of the program.
 
 ---
 
-Placeholder distractor 3
+It is a programming model that applies to all programming problems to solve challenges.
 
 #### --answer--
 
-Placeholder answer
+It is a programming model concerned with data.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The following are importances of OOP except ONE, which one?
 
 #### --distractors--
 
-Placeholder distractor 1
+Achieves security.
 
 ---
 
-Placeholder distractor 2
+Enhances code reusability.
 
 ---
 
-Placeholder distractor 3
+It makes a program flexible.
 
 #### --answer--
 
-Placeholder answer
+It makes the code simple.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following statements is true for the OOP?
 
 #### --distractors--
 
-Placeholder distractor 1
+Multiple instances of objects cannot exist.
 
 ---
 
-Placeholder distractor 2
+OOP can be applied in all cases.
 
 ---
 
-Placeholder distractor 3
+Procedural programs are more secure than OOP programs.
 
 #### --answer--
 
-Placeholder answer
+OOP programs are much larger and more complex to create.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The following are concepts of OOP except ONE. Which one?
 
 #### --distractors--
 
-Placeholder distractor 1
+Polymorphism.
 
 ---
 
-Placeholder distractor 2
+Encapsulation.
 
 ---
 
-Placeholder distractor 3
+Inheritance.
 
 #### --answer--
 
-Placeholder answer
+Data destruction.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which statement is FALSE for OOP in Javascript language?
 
 #### --distractors--
 
-Placeholder distractor 1
+Classes are absent in traditional Javascript language.
 
 ---
 
-Placeholder distractor 2
+Objects inherit from other objects.
 
 ---
 
-Placeholder distractor 3
+Constructors initialize objects.
 
 #### --answer--
 
-Placeholder answer
+Methods can be public, protected, or private.
```

Comment:
It may be worth noting that private methods are part of the specification now.

Reply:
Fixing this by cob today